Gateway/Response/PaymentPosCloudHandler.php

```diff
@@ -13,23 +13,31 @@
 namespace Adyen\Payment\Gateway\Response;
 
 use Adyen\AdyenException;
+use Adyen\Payment\Helper\Quote;
 use Adyen\Payment\Helper\Vault;
 use Adyen\Payment\Logger\AdyenLogger;
-use Adyen\Payment\Model\Order\Payment;
+use Magento\Sales\Model\Order;
+use Magento\Sales\Model\Order\StatusResolver;
```

Comment:
```suggestion
use Magento\Sales\Model\Order\StatusResolver;
use Adyen\Payment\Helper\PaymentResponseHandler;
```

Adding reference class for the following comment.